### PR TITLE
Consistent license message for all the go files

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Consistent license message for all the go files

#### Ticket Link
[MM-20637](https://mattermost.atlassian.net/browse/MM-20637)

### CODE REVIEW

File: services/filesstore/mocks/FileBackend.go [hunk 1]
```diff
@@ -4,9 +4,12 @@
 
 package mocks
 
-import io "io"
-import mock "github.com/stretchr/testify/mock"
-import model "github.com/mattermost/mattermost-server/model"
+import (
+	io "io"
+
```
Comment (catalintomai) [inline services/filesstore/mocks/FileBackend.go]: Intended to be part of the change?

Comment (jespino) [reply services/filesstore/mocks/FileBackend.go]: Actually, wasn't intended, probably my auto go fmt execution did that.

Comment (jespino) [reply services/filesstore/mocks/FileBackend.go]: Fixed that

**Comment by jespino** [comment]
@hanzei, I've created a go vet extension that checks that and I going to add it to the ci to enforce it

**Comment by jespino** [comment]
@hanzei the go vet extension is already there, is more robust and we going to make more checks related to the code there. Is just more reliable to use ast code analysis than pattern matching. In my next PR related to this I going to remove the script entirely and add the go vet checks for mattermost to the CI.

**Comment by jespino** [comment]
@hanzei fixed, and the `app/busy_test.go` too.

**Pull request merged**